applications/nrf5340_audio/src/modules/Kconfig

```diff
@@ -200,4 +200,35 @@ source "subsys/logging/Kconfig.template.log_config"
 
 endif # SD_CARD_PLAYBACK
 
+menuconfig NRF5340_AUDIO_SD_CARD_LC3_STREAMER
+	bool "Audio SD Card LC3 Streamer"
+	depends on NRF5340_AUDIO_SD_CARD_LC3_FILE
+	select EXPERIMENTAL
+	default n
```

Comment:
not going to nack this but `default n` should not be here, that is the default, please remove if updating in future and do not include if adding additional configs in future